### DISCUSSION TITLE
Watch Together functionality

### DISCUSSION THREAD

**Discussion by mvanbaak** [pannal/plex-for-kodi#161] [Ideas]
In a lot of plex apps, if you are invited, you get an extra hub on the homescreen with sessions where you were invited (or the ones you started) for watch together.

The option menu for a media item includes a watch together link (next to the mark watched etc) which brings up a screen where you can invite your plex friends to start a session to watch together.

Support article about it: https://support.plex.tv/articles/watch-together/

I know this is probably a huge thing, but I wanted to drop it here as a feature request and get opinions on it.

**Comment by pannal** [comment]
AFAIK there's no official API for that available. I'm waiting for the documentation about this feature from Plex.

It's on my list.

**Reply by avrignaud** [reply]
Ironically just asked for this same feature on the PM4K forum and then found this. <sigh> Just wanted to +1 here in case it helps! I think the main change since OP requested the feature is that Plex now no longer supports the feature in the new client apps. Details can be found in the support article above. Thanks for considering!
